droidlet/interpreter/robot/tasks.py

```diff
@@ -5,17 +5,21 @@
 import time
 import logging
 import numpy as np
+import math
 
 from droidlet.interpreter.task import BaseMovementTask
 from droidlet.memory.memory_nodes import TaskNode
 from droidlet.interpreter.task import Task
-
+from droidlet.memory.robot.loco_memory import DetectedObjectNode
 from droidlet.interpreter.robot.objects import DanceMovement
 
 from droidlet.lowlevel.locobot.locobot_mover_utils import (
     get_move_target_for_point,
+    get_step_target_for_move,
     ARM_HEIGHT,
+    CAMERA_HEIGHT,
     get_camera_angles,
+    ExaminedMap,
```

Comment:
Can we put this in memory, even as a raw object?  sorry I should have reviewed the last PR...  doesn't need to be well integrated at this point if you want to keep it scrappy; and the code doesn't need to be in the loco_memory.py file (it can be imported from a file in the memory directory)